src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
You mix up things now.
With the latter, how do you share a fresh PluginMountPoint-constant that is added by a plugin at runtime between the plugins? It's not possible.. but it was possible (regression) and it should be possible.

Looks similiar with one imporant difference, it's shared by intention:
```
const {PluginHelper, PluginMountPoints} = window.marathonPluginInterface;
```